android/lib/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.29.2'
-  compile "com.google.android.gms:play-services-base:9.2.0"
-  compile 'com.google.android.gms:play-services-maps:9.2.0'
+  compile 'com.facebook.react:react-native:+'
+  compile "com.google.android.gms:play-services-base:8.4.0"
+  compile 'com.google.android.gms:play-services-maps:8.4.0'
```

Comment:
Ok that's pretty bad. Yeah I don't quite get why this versioning is so problematic to people but I guess at this point it might be better to follow the RN standard then to avoid any more churn.
for GPS we were seeing this crash below a lot and it seems to be a bug on 8.4.0 with Marshmallow devices. Upgrading to 9.2.1 fixed it:

```
java.lang.IllegalArgumentException: Can only use lower 16 bits for requestCode
        at android.support.v4.app.BaseFragmentActivityEclair.checkForValidRequestCode(BaseFragmentActivityEclair.java:64)
        at android.support.v4.app.BaseFragmentActivityEclair.startIntentSenderForResult(BaseFragmentActivityEclair.java:45)
        at android.support.v4.app.FragmentActivity.startIntentSenderForResult(FragmentActivity.java:75)
        at com.google.android.gms.common.ConnectionResult.startResolutionForResult(Unknown:-1)
        at com.google.android.gms.common.api.internal.zzw$zzb.run(Unknown:-1)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:158)
        at android.app.ActivityThread.main(ActivityThread.java:7224)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```